test/tests/functional/pbs_dup_acc_log_for_resv.py

```diff
@@ -378,18 +368,16 @@ def test_multinode_standing_reservation(self):
         # Restart server
         self.server.restart()
 
-        # Verify accounting log,no duplicate log  after server restart
+        # Verify accounting log,no duplicate log after server restart
         m = self.server.accounting_match(
             msg='.*B;' + rid, id=rid, n='ALL', allmatch=True, regexp=True)
         self.assertNotEqual(m, None)
         self.assertEqual(len(m), 1)
 
-        a = {'reserve_state': (MATCH_RE, "RESV_CONFIRMED|2"),
-             'reserve_index': (MATCH_RE, "2")}
+        a = {'reserve_state': (MATCH_RE, "RESV_CONFIRMED|2")}
```

Comment:
It was working fine , but I think you requested to remove the expect on  reserve_index so I removed .
Let me know if I misunderstood your comment on reserve_index :)
"I don't think you need to match RE on this one. You're just matching the string of "1"" 

Even after I removed the reserve_index from expect , test is working fine  and achieving the objective